model.py

```diff
@@ -389,7 +396,8 @@ def from_scraper(cls, type, scraper, region_id):
                 scraper.get_name(),
                 scraper.get_players(),
                 scraper.get_matches(),
-                regions)
+                regions,
+                scraper.get_url())
```

Comment:
I think this should be url=scraper.get_url(); right now I think it is assigning this to alias_to_id_map so urls are not showing up.